3rd_party/3rd_party.cmake

```diff
@@ -220,53 +224,32 @@ function(install_libs _target _source_dir _prefix _postfix)
   endif()
 endfunction()
 
-function(third_party _arg)
-  if("${_arg}" STREQUAL "check")
-    set(INSTALL_DIR "")
```

Comment:
This didn't work at all any more, so I've removed it and simplified the rest of the file as a result.

This file is best reviewed ignoring whitespace differences.